examples/nextjs-flowchart/pages/_app.tsx

```diff
@@ -0,0 +1,8 @@
+import "@/styles/globals.css";
```

Comment:
We don't use TS paths in many places (anywhere even maybe?) so it'd be in favor of not using them here, especially if it's just used here and is synonymous to `./styles/globals.css`.

Reply:
Whoops! Didn't notice this, must be the new Next.js default